packages/blockchain-wallet-v4-frontend/src/modals/Interest/AccountSummary/index.tsx

```diff
@@ -47,13 +48,16 @@ class AccountSummaryContainer extends PureComponent<Props> {
   render () {
     const { data } = this.props
     return data.cata({
-      Success: val => (
-        <AccountSummary
-          {...val}
-          {...this.props}
-          handleDepositClick={this.handleDepositClick}
-        />
-      ),
+      Success: val =>
+        val.walletCurrency === 'TWD' || val.walletCurrency === 'CLP' ? (
```

Comment:
small suggestion we might move `'TWD'` and `'CLP'` to be in constant.

Reply:
Thanks for the suggestion - moved it out to a constant.